app/main/routes.py

```diff
@@ -205,18 +205,25 @@ def search():
     )
     page = int(request.args.get("page", 1))
     filters = {"query": query}
+    sorting_orders = build_sorting_orders(request.args)
 
     if query:
-        fuzzy_search_query = build_fuzzy_search_query(query)
+        query = build_fuzzy_search_query(
```

Comment:
I'd suggest against reusing query for the sqlalchemy query object when it was also used as the query string.

Maybe we should rename query on line 202 to query_string to differentiate it if we don't want to use fuzzy_search_query for the query object.

Reply:
renamed query to fuzzy_search_query as how it was before to avoid confusion